lib/v2/cankaoxiaoxi/maintainer.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
-    '/news/:category': ['yuxinliu-alex'],
+    '/column/:id?': ['yuxinliu-alex', 'nczitzk'],
+    '/news/:id?': ['yuxinliu-alex', 'nczitzk'],
```

Comment:
```suggestion
```
No need to preserve the old path since none of the old parameters can work with the new path